notebooks/v1/ch-machine-learning/machine-learning-qiskit-pytorch.ipynb

```diff
@@ -160,10 +160,12 @@
     "    def run(self, thetas):\n",
     "        t_qc = transpile(self._circuit,\n",
     "                         self.backend)\n",
-    "        qobj = assemble(t_qc,\n",
-    "                        shots=self.shots,\n",
-    "                        parameter_binds = [{self.theta: theta} for theta in thetas])\n",
-    "        job = self.backend.run(qobj)\n",
+    "        # Using a qobj for run() is deprecated as of qiskit-aer 0.9.0.\n",
+    "        # Therefore the transpiled qc (t_qc) is directly passed to the run method.\n",
+    "        # qobj = assemble(t_qc,\n",
+    "        #                 shots=self.shots,\n",
+    "        #                 parameter_binds = [{self.theta: theta} for theta in thetas])\n",
```

Comment:
```suggestion
```

You can remove the comment as the user doesn't need to see the old code.

Reply:
will do